app/server/views/_index_table_of_contents.html.erb

```diff
@@ -0,0 +1,25 @@
+<section class = "l-collection-modules l-book-modules l-padding-60 l-block-wrapper l-block--1080">
+  <div class="l-collection-modules__wrapper">
+    <div class="l-collection-modules__module-episodes">
+      
+     <% book.sections.each do |section| %>
+      <h2><%= section.title %></h2>
+        <div class = "c-tutorial-episode">
+          <div>
+          <ul>
+            <h4>
```

Comment:
I think it's a bit weird having multiple `<li>` tags inside an `<h4>` when they're on multiple lines.

I reckon it'd make more sense to me (semantically) if each `<a>` tag were wrapped in `<h4>`. It might not even need to be `<h4>`, and could be done through some styling.

Reply:
I've gotten rid of the <ul>, it might make a bit more sense now..